test/global_hotkey.tests.ts

```diff
@@ -202,7 +202,7 @@ describe('GlobalHotkey', () => {
 
         GlobalHotkey.register(identity, hotkey, spy);
         //we simulate a window close.
-        ofEvents.emit(route.window('closed'), identity);
+        ofEvents.emit(route.window('closed', '*'), identity);
```

Comment:
why did this need to change (just asking)?

Reply:
The only place we were listening for `route.window('closed')` (with no subtopic/uuid/name) was the subscription manager, so I made it listen to `route.window('closed`, '*') which automatically get's re-emitted on any `route.window('closed', someId)`. That means to simulate a window close we needed to add the asterisk here to match.

Reply:
also, looks like [this place](https://github.com/HadoukenIO/core/blob/develop/src/browser/external_window_event_adapter.js#L136) needs the `*` emit no? @pbaize

Reply:
I don't think so, looks like the listener for that is here https://github.com/pbaize/core/blob/b1495d1065b9f31404576ab424add869f247e80a/src/browser/api/window.js#L508

Reply:
👍 you're right